psalm.xml

```diff
@@ -70,6 +70,7 @@
         <InvalidReturnType>
             <errorLevel type="suppress">
                 <file name="src/Persistence/Fake/FakeResetPasswordInternalRepository.php"/>
+                <file name="src/Command/ResetPasswordRemoveExpiredCommand.php" />
```

Comment:
Moving it here would mean that Psalm no longer reports warnings on *any* InvalidReturnTypes for any code in this class, not just for the `execute()` method (it might be the only method, but you get the idea). Is the .xml file considered a better practice?

Reply:
Going to play around with this for a bit to see if I can come up with a better solution. Else, don't merge and will close PR.. 

For this method as is, its not a big deal either way, but disabling an inspection for the entire class is probably not a get precedent to set if it can be avoided.